lib/src/components/NxToggle/NxToggle.scss

```diff
@@ -71,6 +71,7 @@ $nx-toggle-width: 44px;
   margin: 0 0 0 var(--nx-spacing-3x);
   width: 0;
   -moz-appearance: none;
+  outline: 0;
```

Comment:
Does `none` work? That feels cleaner to me

Reply:
> Does `none` work? That feels cleaner to me

Changed. 
a617179ea71e9e80b0be56c9a4b0a0c3a88d6fa7